superset/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -98,12 +105,12 @@ class QueryTable extends React.PureComponent {
       q.started = moment(q.startDttm).format('HH:mm:ss');
       q.querylink = (
         <div style={{ width: '100px' }}>
-          <a
-            href={this.getQueryLink(q.dbId, q.sql)}
-            className="btn btn-primary btn-xs"
+          <button
+            className="btn btn-link btn-xs"
+            onClick={this.openQuery.bind(this, q.schema, q.dbId, q.sql)}
```

Comment:
`openQuery` should be called with the args in this order `openQuery(dbId, schema, sql)`